src/ensembl/src/shared/components/panel/Panel.scss

```diff
@@ -0,0 +1,45 @@
+@import 'src/styles/common';
+
+.panel {
+  position: relative;
+  min-height: 100px;
+  border-radius: 5px;
+  width: 600px;
+  max-width: 100%;
+  background: $white;
+  box-shadow: 1px 1px 4px $shadow-color;
+}
+
+.header {
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
+  background-color: $light-grey;
+  box-shadow: 0px 1px 3px $medium-light-grey;
+  height: 34px;
+  padding: 4px 0 4px 10px;
```

Comment:
There's a 10px padding on the left, but no symmetrical padding on the right? What happens if the heading occupies the whole header space?

Reply:
Change the padding to be symmetrical.

>What happens if the heading occupies the whole header space?

I am not sure what is best to do here.
Adding overflow seems to be causing the problem we had here:
https://github.com/Ensembl/ensembl-client/pull/160